fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -48,6 +48,18 @@ def self.ci?
       return false
     end
 
+    def self.unix?
```

Comment:
Not sure what this is useful for in this day and age, I think we should remove it.